src/commands/apps/lib/app-environment-validation.ts

```diff
@@ -0,0 +1,10 @@
+export const APP_ENVIRONMENT_VALIDATIONS = {
+  maxLength: {
+    rule: 100,
+    errorMessage: `environment-app length must be 100 characters or less`
+  },
+  matchRegexp: {
+    rule: /^$|^[A-Z0-9][a-z0-9]*$/,
+    errorMessage: `environment-app must be a single word starting with a capital letter or number, followed by lowercase`
+  }
+};
```

Comment:
Shouldn't we validate against the only accepted values: "Alpha, Beta, Production, Store, Enterprise"?

The error message should be explicit about what values are expected.

Reply:
See above comment - custom values are permitted, those are just suggestions :D